Add Turing Way CC (#1722)
betterscientificsoftware/bssw.io: CuratedContent/TuringWay.md

# The Turning Way: Reproducible and Ethical Science. 
<!--deck text start-->
An online handbook to reproducible, ethical, and collaborative computational and data science.
<!--deck text end-->

#### Contributed by [Jed Brown](https://github.com/jedbrown), [Mark C. Miller](https://github.com/markcmiller86 "Mark C. Miller GitHub Profile")
#### Publication date: June 22, 2023

Resource information | Details
:--- | :--- 

> Article title  | [The Turing Way](https://the-turing-way.netlify.app/index.html), [DOI (most up to date)](http://doi.org/10.5281/zenodo.3332807)

Reviewer: That's the DOI for the illustrations, this is the Zenodo concept DOI
```suggestion
Article title  | [The Turing Way](https://the-turing-way.netlify.app/index.html), [DOI (most up to date)](https://doi.org/10.5281/zenodo.3233853)
```

> Authors | The Turing Way Community, Becky Arnold, Louise Bowler, Sarah Gibson, Patricia Herterich, Rosie Higman, … Kirstie Whitaker. (2019, March 25). The Turing Way: A Handbook for Reproducible Data Science (Version v0.0.4). Zenodo. 

Reviewer: ```suggestion
Authors | The Turing Way Community, (2022, Dec 21). The Turing Way: A handbook for reproducible, ethical and collaborative research. Zenodo. 
```
I think this from their GitHub readme is more future-proof way to cite. I'll see about updating the guidance on the website.
https://github.com/alan-turing-institute/the-turing-way#citing-the-turing-way

Length | Multi-Chapter, online handbook
Focus | Reproducible, ethical and collaborative data science

This is a wonderful handbook to reproducible, ethical, and collaborative computational and data science.
It is an online collaborative project that brings together the expertise and experiences of various contributors in the field of data science and research.
It contains impactful illustrations and is written in a consistent, approachable style.
Although it does not have an HPC focus and some of the details are specific to biomedical applications, the chapters on Reproducible Research, Project Design, Communication, Collaboration, and Ethical Research are largely applicable to HPC/CSE.

> *The Turing Way* project is open source, open collaboration, and community-driven.
> It involves and supports a diverse community of contributors to make data science accessible, comprehensible and effective for everyone.
> It's goal is to provide all the information that researchers and data scientists in academia, industry and the public sector need to ensure that the projects they work on are easy to reproduce and reuse.

Reviewer: I was thinking of having this section quoted and then it can still use "our" like in the original. You can choose which version to accept.
```suggestion
*The Turing Way* project is open source, open collaboration, and community-driven.
It involves and supports a diverse community of contributors to make data science accessible, comprehensible and effective for everyone.
Its goal is to provide all the information that researchers and data scientists in academia, industry and the public sector need to ensure that the projects they work on are easy to reproduce and reuse.
```

Reviewer: ```suggestion
> *The Turing Way* project is open source, open collaboration, and community-driven.
It involves and supports a diverse community of contributors to make data science accessible, comprehensible and effective for everyone.
Our goal is to provide all the information that researchers and data scientists in academia, industry and the public sector need to ensure that the projects they work on are easy to reproduce and reuse.
```

All stakeholders, including researchers, software engineers, project leaders and funding teams, are encouraged to use *The Turing Way* to understand their roles and responsibility of reproducibility in data science.

> The moonshot goal of *The Turing Way* is to make reproducibility *too easy not to do*.

Reviewer: Possibly `>` quote this since it's their words and written in their voice?

Author: Done

> # Reproducible and ethical science. 

Reviewer 2: Is this the best title for this CC article?  Perhaps something like "The  Turning Way: Reproducible and Ethical Science"?

Reviewer 2: The title "Reproducible and ethical science" would make me thing this is a deep article on reproducability and ethics and would site may sources on the topic.  I think that adding the name of the article "The Turing Way" would eliminate such confusion.

<!--deck text start-->
An on line handbook to reproducible, ethical, and collaborative computational and data science.
<!--deck text end-->

#### Contributed by [Jed Brown](https://github.com/jedbrown), [Mark C. Miller](https://github.com/markcmiller86 "Mark C. Miller GitHub Profile")
#### Publication date: June 22, 2023

Resource information | Details
:--- | :--- 
Article title  | [The Turing Way](https://the-turing-way.netlify.app/index.html)

> Authors | The Turing Way Community, Becky Arnold, Louise Bowler, Sarah Gibson, Patricia Herterich, Rosie Higman, … Kirstie Whitaker. (2019, March 25). The Turing Way: A Handbook for Reproducible Data Science (Version v0.0.4). Zenodo. 

Reviewer: > Please visit the [DOI link](https://doi.org/10.5281/zenodo.3233853) though to get the most recent version - the one above is not automatically generated and may be out of date.

The current version is 1.0.2.

Author: Ok, added DOI link with a note about out of date. That version is much more nicely formatted though.

Length | Multi-Chapter, on line handbook
Focus | Reproducible, ethical and collaborative data science

> This is a wonderful handbook to reproducible, ethical, and collaborative computational and data science.

Reviewer 2: "wonderful" seems a little strong.  Perhaps more techy words like "impressive" or "solid"?

Author: Wonderful was @jedbrown actual words. I didn't want to adjust it.

It is an online collaborative project that brings together the expertise and experiences of various contributors in the field of data science and research.

> It contains lovely illustrations and is written in a consistent, approachable style.

Reviewer 2: Perhaps replace "lovely" with a more techy word like "useful" or "impactful"?

Reviewer: Why? When you look at the illustrations, how would you describe them that doesn't lump them in with every manager's box-and-arrows powerpoint?

Author: Ok, changed that.

Although it does not have an HPC focus and some of the details are specific to biomedical applications, the chapters on Reproducible Research, Project Design, Communication, Collaboration, and Ethical Research are largely applicable to HPC/CSE.

> The book takes its name from Alan Turing, a pioneering mathematician and computer scientist, and aims to embody his values of openness, inclusivity, and rigor. It provides practical advice and best practices for conducting data science projects in a reproducible manner, emphasizing the importance of transparency, openness, and accessibility.

Reviewer: This reads like ChatGPT-generated bullshit in the sense that the authors don't use these terms to explain the name. I would guess "pioneering" (a colonialism that at least some of the Turing Way contributors would not select) comes from this bio https://www.turing.ac.uk/about-us. The subsequent paragraphs are wordy and repetitive compared to just quoting the preface of the book, and don't have the copyright catastrophe of ChatGPT. Could we please use quote this instead?

> The Turing Way project is open source, open collaboration, and community-driven. We involve and support a diverse community of contributors to make data science accessible, comprehensible and effective for everyone. Our goal is to provide all the information that researchers and data scientists in academia, industry and the public sector need to ensure that the projects they work on are easy to reproduce and reuse.
>
> All stakeholders, including researchers, software engineers, project leaders and funding teams, are encouraged to use The Turing Way to understand their roles and responsibility of reproducibility in data science. You can inspect our resources on [GitHub](https://github.com/alan-turing-institute/the-turing-way), contribute to the project as described in our [contribution guidelines](https://github.com/alan-turing-institute/the-turing-way/blob/main/CONTRIBUTING.md) and re-use all materials ([see the License](https://github.com/alan-turing-institute/the-turing-way/blob/main/LICENSE.md)).

Reviewer: Alternate quote from their GitHub readme:

> Started in 2019 as a lightly opinionated guide to data science, The Turing Way has since expanded into a series of guides on Reproducible Research, Project Design, Communication, Collaboration and Ethical Research. Each guide offers chapters on a range of topics covering best practices, guidance and recommendations. These chapters have been co-authored by contributors who are students, researchers, educators, community leaders, policy-makers and professionals from diverse backgrounds, lived experiences and domain knowledge.
>
> Our moonshot goal is to make reproducibility "too easy not to do".

Author: I used the first but kept the moonshot remark too.
